VideoLocker/res/layout/activity_login.xml

```diff
@@ -59,47 +59,66 @@
                         android:paddingLeft="44dp"
                         android:paddingRight="44dp">
 
-                        <EditText
-                            android:id="@+id/email_et"
-                            style="@style/edit_text_style"
-                            android:hint="@string/email_username"
-                            android:inputType="textEmailAddress"
-                            android:maxLength="100" />
-
-                        <EditText
-                            android:id="@+id/password_et"
-                            style="@style/edit_text_style"
-                            android:layout_marginTop="8dp"
-                            android:hint="@string/password"
-                            android:inputType="textPassword"
-                            android:maxLength="100" />
+                        <android.support.design.widget.TextInputLayout
+                            android:id="@+id/usernameWrapper"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:textColorHint="@color/grey_text">
+
+                            <EditText
```

Comment:
Also, I believe the `hint` is supposed to be set on `TextInputLayout`, not on the EditText itself.